src/editor/actions/console-message-actions.js

```diff
@@ -1,45 +1,37 @@
-import createHistoryItem from "../../shared/utils/create-history-item";
```

Comment:
got rid of `createHistoryItem` in favor of doing it's job of making sure random historyIds are always added here and at the similar point in the eval-frame